lib/content/audio/track_change.ex

```diff
@@ -71,6 +71,20 @@ defmodule Content.Audio.TrackChange do
       end
     end
 
+    def to_tts(%Content.Audio.TrackChange{} = audio) do
+      train = PaEss.Utilities.train_description(audio.destination, audio.route_id)
+
+      platform =
+        case audio.berth do
+          "70196" -> "B"
+          "70197" -> "C"
+          "70198" -> "D"
+          "70199" -> "E"
```

Comment:
I see the existing logic maps to a few different take ids depending on the destination. Do those takes just call out the branch letters or do they contain more detail that we would need to include here? I can't recall from memory what those messages sound like in practice.